FreeRTOS-Plus/Demo/coreSNTP_Windows_Simulator/SNTPClientTask.c

```diff
@@ -1158,7 +1158,7 @@ SntpStatus_t addClientAuthCode( SntpAuthContext_t * pAuthContext,
 SntpStatus_t validateServerAuth( SntpAuthContext_t * pAuthContext,
                                  const SntpServerInfo_t * pTimeServer,
                                  const void * pResponseData,
-                                 uint16_t responseSize )
+                                 size_t responseSize )
```

Comment:
This function is passed as the implementation of this signature - https://github.com/FreeRTOS/coreSNTP/blob/main/source/include/core_sntp_client.h#L331. It cannot be changed without changing the interface signature.